powersimdata/scenario/execute.py

```diff
@@ -371,20 +371,13 @@ def prepare_profile(self, kind, profile_as=None):
             print(
                 f"Writing scaled {kind} profile in {server_setup.LOCAL_DIR} on local machine"
             )
-            file_name = "%s_%s.csv" % (self._scenario_info["id"], kind)
+            file_name = "%s_%s.csv" % (self.scenario_id, kind)
             profile.to_csv(os.path.join(server_setup.LOCAL_DIR, file_name))
 
             self._data_access.move_to(
                 file_name, self.REL_TMP_DIR, change_name_to=f"{kind}.csv"
             )
         else:
-            from_dir = posixpath.join(
-                self.server_config.execute_dir(),
-                f"scenario_{profile_as}",
-            )
-            to_dir = posixpath.join(
-                self.server_config.execute_dir(), self.scenario_folder
-            )
-            _, _, stderr = self._data_access.copy(f"{from_dir}/{kind}.csv", to_dir)
-            if len(stderr.readlines()) != 0:
-                raise IOError(f"Failed to copy {kind}.csv on server")
+            from_dir = self._data_access.match_scenario_files(profile_as, "tmp")
+            to_dir = self.TMP_DIR
+            self._data_access.copy(f"{from_dir}/{kind}.csv", to_dir)
```

Comment:
It is kind of confusing to `move to` using relative path but to `copy` with absolute path

Reply:
Agree, I just didn't know a cleaner way to do it. The reason for this is mainly how we use `copy` within `move.py`, where the destination (backup disk) is not relative to the `self.root`. Maybe we can revisit this in another PR.

Reply:
Sounds good